src/python/pants/backend/explorer/browser.py

```diff
@@ -50,5 +55,5 @@ async def get_browser(request: BrowserRequest, open_binary: OpenBinary) -> Brows
 def rules():
     return (
         *collect_rules(),
-        QueryRule(ProcessResult, (Process,)),
+        QueryRule(ProcessResult, (Process, ProcessConfigFromEnvironment)),
```

Comment:
We want the parsing logic to live in Python because us the Target API to determine the values of `ProcessConfigFromEnvironment`. Rust doesn't know how to use the Target API and I don't want to teach it.

Reply:
Understood. But if this `QueryRule` takes an `EnvironmentName`, it will compute the `ProcessConfigFromEnvironment` from it as our other callsites do.

Reply:
Ah okay, I can try that!